modules/@ergonode/ui/src/models/treeAccordion/index.js

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright © Bold Brand Commerce Sp. z o.o. All rights reserved.
+ * See LICENSE for license details.
+ */
+
+export const getSelectedItems = value => value.reduce((prev, curr) => ({
+    ...prev,
+    [curr.id]: true,
+}), {});
+
+export const getSelectedNodesCount = ({
+    value,
+    treeStructure,
+}) => {
+    const nodesCount = {};
+
```

Comment:
no need to break the lines

Reply:
I like to have function declaration  with one break